src/Cohesity.Powershell/Scripts/Recover-CohesityBackupToView.ps1

```diff
@@ -114,4 +114,4 @@ function Recover-CohesityBackupToView {
     }
 }
 
-Recover-CohesityBackupToView
\ No newline at end of file
+Recover-CohesityBackupToView -QOSPolicy "TestAndDev High"
```

Comment:
Remove this ! after you test , remove the last lines ..

Reply:
Fixed.